src/elements/card/src/index.tsx

```diff
@@ -6,14 +6,16 @@ import { ImgixImage } from '@uswitch/trustyle.imgix-image'
 
 interface Props {
   className?: string
-  description: string
+  date?: string
```

Comment:
Should we be calling this `date`?

If it's just a string I wonder if the prop should be more of a description of how it will present that string so it can be used for other data, e.g. `tag="Click me now!"`

Maybe `superScriptHeader` to suggest it appears about the header, or `tagInfo` as it sits next to the tag?

If the intent is for it to always be a date, I think it should be a date object, i.e. `new Date()`, and the formatting should be configurable at a brand/theme level with defaults to something like `dd YYYY mm`

Reply:
I think we should keep the string as a date object would involve configuration. I will change the name of the prop to something more appropriate